build-system/pr-check/experiment-tests.js

```diff
@@ -51,6 +52,7 @@ function pushBuildWorkflow() {
       jobName,
       `${experiment} is expired, misconfigured, or does not exist`
     );
+    reportTestSkipped();
```

Comment:
When called from within a `gulp` task `reportTestSkipped()`, infers the test type from the command line. I don't believe calling it from within `experiment-tests.js` will do anything because this is the job script wrapper and there is no `gulp` task running at this point.

One option is to change `reportAllExpectedTests()` to read the experiment flags and only report  enabled experiments as queued. I'm not sure what the other options are given that this is a unique case where running an experiment is determined based on the config file and not on the build targets.

Reply:
Yep, you're right. It didn't work. Fixed this - PTAL